src/component/MoviesListPaper.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {connect} from 'react-redux';
+import Paper from '@material-ui/core/Paper';
+import Button from '@material-ui/core/Button';
+import FormControlLabel from '@material-ui/core/FormControlLabel';
+
+import Checkbox from '@material-ui/core/Checkbox';
+import { fetchRelatedMovies, setSelectedMovie } from '../store/actions/MovieActions';
+import { Link } from 'react-router-dom';
+class MoviesListPaper extends React.Component{
+    
+    
+    setSelectedMovie =(movie)=>{
+        this.props.setSelectedMovie(movie);
+        this.props.fetchRelatedMovies({id : movie.id});
```

Comment:
setSelectedMovie bi trebalo da selektuje film samo i to je to, a related movies treba da fetch-uje sama komponenta za related Movies, hajde popricacemo o ovome kad stignes. Pisi pa kad stignem da prodjemo organizaciju komponenti

Reply:
^^^